boards/lora-e5-dev/dist/openocd.cfg

```
@@ -1,3 +1,4 @@
 source [find target/stm32wlx.cfg]
+set _TARGETNAME $_CHIPNAME.cpu0
 reset_config trst_only
 $_TARGETNAME configure -rtos auto
```

```suggestion
reset_config trst_only
$_CHIPNAME.cpu0 configure -rtos auto
```

Why not do it like OpenOCD does it and just use the `$_CHIPNAME.cpu0` variable directly, skipping the `_TARGETNAME`?